frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -230,7 +230,10 @@ function FileManagerMenu:setUpdateItemTable()
     self.menu_items.cloud_storage = {
         text = _("Cloud storage"),
         callback = function()
-            local cloud_storage = CloudStorage:new{}
+            local cloud_storage = CloudStorage:new{
+                width = Screen:getWidth(),
```

Comment:
I see, when I glanced at it I assumed it was part of the initialization. But then the solution is surely to make it not remember rather than to make life difficult on people trying to create an instance. ;-)

Sticking the width and height in here instead of where they are ought to do it?
https://github.com/koreader/koreader/blob/3492b8b117e6401f145acf346f6e4be4db2752ad/frontend/apps/cloudstorage/cloudstorage.lua#L32-L39